<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TOPLauncherClass</class>
 <widget class="QMainWindow" name="TOPLauncherClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>670</width>
    <height>450</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Tahoma,9pt SimSun,9pt 宋体</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>TOPLauncher</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
  font: 10pt &quot;Tahoma&quot;, 9pt &quot;SimSun&quot;,  9pt &quot;宋体&quot;;
}

QWidget#centralWidget {
  background: url(:/TOPLauncher/Resources/bg.png);
}

QFrame#frameLogin {
  background-color: rgba(255, 255, 255, 192);
}

QFrame#frameBottom {
  background-color: rgba(255, 255, 255, 192);
}</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="locale">
       <locale language="Chinese" country="China"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget" native="true"/>
          </item>
          <item>
           <widget class="QFrame" name="frameLogin">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>170</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <item>
              <widget class="QGroupBox" name="groupLogin">
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>400</width>
                 <height>150</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Tahoma,9pt SimSun,9pt 宋体</family>
                 <pointsize>10</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="title">
                <string>Login</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QFrame" name="frame_3">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_4">
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout">
                       <item>
                        <layout class="QVBoxLayout" name="verticalLayout_3">
                         <item>
                          <widget class="QLabel" name="labelUsername">
                           <property name="locale">
                            <locale language="English" country="UnitedStates"/>
                           </property>
                           <property name="text">
                            <string>Username:</string>
                           </property>
                           <property name="alignment">
                            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QLabel" name="labelPassword">
                           <property name="locale">
                            <locale language="English" country="UnitedStates"/>
                           </property>
                           <property name="text">
                            <string>Password:</string>
                           </property>
                           <property name="alignment">
                            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                       <item>
                        <layout class="QVBoxLayout" name="verticalLayout">
                         <item>
                          <widget class="QComboBox" name="comboUsername">
                           <property name="locale">
                            <locale language="English" country="UnitedStates"/>
                           </property>
                           <property name="editable">
                            <bool>true</bool>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QLineEdit" name="editPassword">
                           <property name="locale">
                            <locale language="English" country="UnitedStates"/>
                           </property>
                           <property name="maxLength">
                            <number>64</number>
                           </property>
                           <property name="echoMode">
                            <enum>QLineEdit::Password</enum>
                           </property>
                           <property name="clearButtonEnabled">
                            <bool>true</bool>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_3">
                       <item>
                        <spacer name="horizontalSpacer_2">
                         <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                       <item>
                        <widget class="QCheckBox" name="chkRememberPasswd">
                         <property name="locale">
                          <locale language="English" country="UnitedStates"/>
                         </property>
                         <property name="text">
                          <string>Remember Password</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <widget class="QPushButton" name="btnLogin">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>100</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="locale">
                      <locale language="English" country="UnitedStates"/>
                     </property>
                     <property name="text">
                      <string>Start</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                   <widget class="QPushButton" name="btnLanguage">
                    <property name="text">
                     <string>Language...</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btnRegister">
                    <property name="locale">
                     <locale language="English" country="UnitedStates"/>
                    </property>
                    <property name="text">
                     <string>Register</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btnOpenForum">
                    <property name="locale">
                     <locale language="English" country="UnitedStates"/>
                    </property>
                    <property name="text">
                     <string>Forum</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frameBottom">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Tahoma,9pt SimSun,9pt 宋体</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnSettings">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Settings...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelChooseServer">
         <property name="text">
          <string>Select a server:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboServer">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnExit">
         <property name="text">
          <string>Quit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TOPLauncher.qrc"/>
 </resources>
 <connections/>
</ui>
